### PR TITLE
#12832903: user retrieve notifications

### PR DESCRIPTION
**Pull request by bolorundurowb**

### CODE REVIEW

File: controllers/Notifications.js [hunk 1]
```diff
@@ -7,6 +7,42 @@ const OutputFormatters = require('./../utils/OutputFormatters');
...
+      const notificationLastAccessedAt = user.notificationsLastRetrievedAt || new Date(1970, 0, 1);
+
+      const subscriptions = await db.Subscription
+        .find({
+          email: user.email
+        })
+        .select('politician');
+      const subscribedPoliticians = subscriptions.map(x => x.politician);
+
+      const notifications = await db.Notification
```
Comment (andela-abisoye) [inline controllers/Notifications.js]: This looks like it will only retrieve the politician updates and not the political party updates. Any reason why?

Comment (bolorundurowb) [reply controllers/Notifications.js]: Users are not subscribed to political parties currently hence the system.

Comment (bolorundurowb) [reply controllers/Notifications.js]: I could update it and add in all political party notifications as well but we currently do not generate notifications for political parties either

Comment (Saint-lawal) [reply controllers/Notifications.js]: Alright makes sense

**Pull request merged**